Doc/library/syslog.rst

```diff
@@ -72,6 +80,10 @@ The module defines the following functions:
 
    .. audit-event:: syslog.closelog "" syslog.closelog
 
+   .. versionchanged:: 3.12
+      Only the main interpreter is allowed to call. It will raise :exc:`RuntimeError`
+      if subinterpreters call :func:`closelog`.
```

Comment:
```suggestion
      This function is restricted in subinterpreters.
      (Only code that runs in multiple interpreters is affected and
      the restriction is not relevant for most users.)
      This may only be called in the main interpreter.
      It will raise :exc:`RuntimeError` if called in a subinterpreter.
```